.bazelci/presubmit.yml

```diff
@@ -4,12 +4,20 @@ platforms:
     build_targets:
     - "..."
     test_targets:
+    # We are running the rest of test targets remotely only.
+    - "--"
     - "..."
+    - "-//tests/container/..."
+    - "-//tests/docker/..."
   ubuntu1604:
     build_targets:
     - "..."
```

Comment:
lets also remove some of the build targets, with note we're doing them remotely

Reply:
Done.